docs/custom_conf.py

```diff
@@ -146,6 +146,24 @@
 
 ## The following settings override the default configuration.
 
+manpages_url = 'https://manpages.ubuntu.com/manpages/noble/en/man{section}/{page}.{section}.html'
+
+rst_prolog = '''
+.. caution::
+
+   The Packaging and Development guide is currently undergoing a major overhaul
+   to bring it up to date. The current state you are seeing now is a preview of
+   this effort.
+
+   The current version is unstable (changing URLs can occur at any time) and
+   most content is not in properly reviewed yet. Proceed with caution and be
+   aware of technical inaccuracies.
+
+   If you are an experienced packager and would like to contribute, we would
+   love for you to be involved! See :doc:`our contribution page </contribute>`
+   for details of how to join in.
+'''
+
```

Comment:
I would prefer it, because I had to explain multiple times to people that this is not final. Even with the old "Work in progress" banner. The new banner is hopefully of a size that is hard to ignore, but I can also live without it.

Reply:
I would also argue, that many people expect a stable version when selecting `latest`

Reply:
Fair point, and if you've already had people using it then you're right, we should point out that it's still a WIP.